content/blog/2017-11-10-rprofile-mara-averick.md

```diff
@@ -0,0 +1,127 @@
+---
+slug: rprofile-mara-averick
+title: ".rprofile: Mara Averick"
+date: 2017-11-10
+preface: "In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “demystifying the creative and development processes of R community members”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project."
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+categories: blog
+topicid: 921
+tags:
+- R
+- community
+- interviews
+---
+
+<img src="/img/blog-images/2017-11-10-rprofile-mara-averick/mara_averick.png" alt="Mara Averick, Data Nerd At Large" align="left" style="margin: 0px 20px", width="250px">
```

Comment:
I've asked for some advice on this: David Smith pic and text alignment looks perfect now but the preview of Mara's looks too big and the text runs below the pic.

I'm naive, and the only diff I see is slight difference in `style="margin: 0px 20px; width: 250px;"` compared with what you use here.

No action needed on your part now, unless you know an immediate solution. It's my to-do right now.